MultiSource/Benchmarks/Ptrdist/ft/ft.c

```diff
@@ -89,30 +91,30 @@ main(int argc, _Array_ptr<const char*> argv : count(argc) )
   }
 
   if(debug)
-  {
+  _Unchecked {
```

Comment:
Even better, I added a checked scope for `main`, and an Unchecked scope just for the argument manipulation